rpc/src/rpc_service.rs

```diff
@@ -745,7 +745,9 @@ mod tests {
         assert!(!rrm.is_file_get_path("//genesis.tar.bz2"));
         assert!(!rrm.is_file_get_path("/../genesis.tar.bz2"));
 
-        assert!(!rrm.is_file_get_path("/snapshot.tar.bz2")); // This is a redirect
+        // These two are redirects
+        assert!(!rrm.is_file_get_path("/snapshot.tar.bz2"));
+        assert!(!rrm.is_file_get_path("/incremental-snapshot.tar.bz2"));
```

Comment:
It'd be nice to update these to `zstd` since `bz2` would no longer be supported, but for legacy reasons, I think we need to keep these as-is; thoughts ?

Reply:
Removing these will cause ecosystem breakage ([e.g.](https://github.com/mvines/sosh/blob/18b1691e212357c3d51514a8ec897380bfc9668f/bin/fetch-snapshot.sh#L49-L57)), I'd leave them as is for now. It's a little bit 🤡 but not really harmful.
We've also been kicking around the idea of a new snapshot format and if/when that happens perhaps that's an opportunity to fix up these redirects as we'll then have better knowledge of what the future of snapshots is expected to be

Reply:
Thanks for the confirmation - I noticed we use it in some of our scripts and [docs](https://docs.solana.com/validator/best-practices/operations
) it so assumed it'd be too disruptive to change, but the nitpicker in me still wanted to ask. We'll leave it as-is

> We've also been kicking around the idea of a new snapshot format and if/when that happens perhaps that's an opportunity to fix up these redirects as we'll then have better knowledge of what the future of snapshots is expected to be

Sounds good